src/Common/src/Internal/Text/Utf8String.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Internal.Text
 {
-    public struct Utf8String : IEquatable<Utf8String>
+    public struct Utf8String : IEquatable<Utf8String>, IComparable
```

Comment:
Would it make sense to implement `IComparable<Utf8String>` here instead to avoid boxing?

Reply:
Yep; done